### ISSUE TITLE
Would u like to share the requirements, tks!

### ISSUE THREAD

**Issue by smartcai** [Vafali/Development-of-Deep-Reinforcement-Learning-policy-in-a-Parking-simulated-environment#1]
Would u like to share the requirements, tks!

**Comment by smartcai** [comment]
The following enviroment works well.
python=3.7
Package                      Version
---------------------------- -----------
absl-py                      2.1.0
astunparse                   1.6.3
cachetools                   5.3.3
certifi                      2024.2.2
charset-normalizer           3.3.2
cycler                       0.11.0
flatbuffers                  24.3.25
fonttools                    4.38.0
gast                         0.4.0
google-auth                  2.29.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
grpcio                       1.62.2
h5py                         3.8.0
idna                         3.7
importlib-metadata           6.7.0
keras                        2.11.0
kiwisolver                   1.4.5
libclang                     18.1.1
Markdown                     3.4.4
MarkupSafe                   2.1.5
matplotlib                   3.5.3
numpy                        1.21.6
oauthlib                     3.2.2
opencv-contrib-python        3.4.3.18
opt-einsum                   3.3.0
packaging                    24.0
pandas                       1.3.5
Pillow                       9.5.0
pip                          24.0
protobuf                     3.19.6
pyasn1                       0.5.1
pyasn1-modules               0.3.0
pyparsing                    3.1.2
python-dateutil              2.9.0.post0
pytz                         2024.1
requests                     2.31.0
requests-oauthlib            2.0.0
rsa                          4.9
setuptools                   69.0.3
six                          1.16.0
tensorboard                  2.11.2
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.11.0
tensorflow-estimator         2.11.0
tensorflow-intel             2.11.0
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.3.0
typing_extensions            4.7.1
urllib3                      2.0.7
Werkzeug                     2.2.3
wheel                        0.42.0
wrapt                        1.16.0
zipp                         3.15.0

**Comment by Vafali** [comment]
Hi @smartcai , 

Good luck. 


anyio==3.6.1
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
attrs==21.4.0
Babel==2.10.1
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.0
carla-0.9.13-cp38-cp38-linux_x86_64.whl; sys_platform == 'linux'
carla==0.9.13; sys_platform == 'win32'
certifi==2022.5.18.1
cffi==1.15.0
charset-normalizer==2.0.12
cloudpickle==2.0.0
collision
colorama==0.4.4
cycler==0.11.0
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
deprecation==2.1.0
entrypoints==0.4
executing==0.8.3
fastjsonschema==2.15.3
fonttools==4.33.3
future==0.18.2
gym==0.21.0
gym-notices==0.0.6
idna==3.3
imageio==2.19.2
importlib-metadata==4.13.0
importlib-resources==5.7.1
ipykernel==6.13.0
ipython==8.3.0
ipython-genutils==0.2.0
ipywidgets==7.7.0
jedi==0.18.1
Jinja2==3.1.2
json5==0.9.8
jsonschema==4.5.1
jupyter-client==7.3.1
jupyter-core==4.10.0
jupyter-packaging==0.12.0
jupyter-server==1.17.0
jupyterlab==3.4.2
jupyterlab-pygments==0.2.2
jupyterlab-server==2.14.0
jupyterlab-widgets==1.1.0
kiwisolver==1.4.2
MarkupSafe==2.1.1
matplotlib==3.5.2
matplotlib-inline==0.1.3
mistune==0.8.4
nbclassic==0.3.7
nbclient==0.6.3
nbconvert==6.5.0
nbformat==5.4.0
nest-asyncio==1.5.5
networkx==2.8.2
notebook==6.4.11
notebook-shim==0.1.0
numpy==1.22.4
open3d
packaging==21.3
pandas==1.4.2
pandocfilters==1.5.0
parso==0.8.3
pickleshare==0.7.5
Pillow==9.1.1
prometheus-client==0.14.1
prompt-toolkit==3.0.29
psutil==5.9.0
pure-eval==0.2.2
pycparser==2.21
pygame==2.1.2
Pygments==2.12.0
pyparsing==3.0.9
pyrsistent==0.18.1
python-dateutil==2.8.2
pytz==2022.1
PyWavelets==1.3.0
pyzmq==23.0.0
requests==2.27.1
scikit-image==0.19.2
scipy==1.8.1
Send2Trash==1.8.0
six==1.16.0
sniffio==1.2.0
soupsieve==2.3.2.post1
stable-baselines3[extra]==1.6.2
stack-data==0.2.0
tensorboard
terminado==0.15.0
tifffile==2022.5.4
tinycss2==1.1.1
tomlkit==0.10.2
tornado==6.1
traitlets==5.2.1.post0
typing_extensions==4.2.0
urllib3==1.26.9
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.3.2
widgetsnbextension==3.6.0
zipp==3.8.0
